### DISCUSSION TITLE
Custom forum icons

### DISCUSSION THREAD

**Discussion by Seathin** [NamelessMC/Nameless#2100] [Ideas]
It would be cool if you could change these icons 
![image](https://user-images.githubusercontent.com/78930928/107688714-9f71a080-6cb0-11eb-889f-b26322e707f0.png) to .png icons

**Comment by Seathin** [comment]
Forum

**Reply by BaxAndrei** [reply]
This is already posible.
![image](https://user-images.githubusercontent.com/12484505/107698676-90ddb600-6cbd-11eb-97ea-e5c4da14f31a.png)
